implementation/domain/logic/jbb-system-impl/src/test/java/org/jbb/system/impl/session/SessionServiceImplTest.java

```diff
@@ -0,0 +1,194 @@
+package org.jbb.system.impl.session;
+
+
+import com.google.common.collect.Maps;
+
+import org.jbb.lib.core.security.SecurityContentUser;
+import org.jbb.lib.mvc.repository.JbbSessionRepository;
+import org.jbb.system.api.model.session.UserSession;
+import org.jbb.system.impl.base.properties.SystemProperties;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextImpl;
+import org.springframework.session.ExpiringSession;
+import org.springframework.session.MapSession;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class SessionServiceImplTest {
+
+    @Mock
+    private JbbSessionRepository jbbSessionRepository;
+
+    @Mock
+    private SystemProperties systemProperties;
+
+    private SessionServiceImpl sessionService;
+
+    @Before
+    public void init(){
+        when(systemProperties.sessionMaxInActiveTime()).thenReturn(3600);
+        this.sessionService = new SessionServiceImpl(jbbSessionRepository,systemProperties);
```

Comment:
Performance? I don't think so. I was thinking about more readability and conciseness (you don't need to use constructor explicit and inject mocks manually)